app/assets/javascripts/angular/controllers/dashboard_ctrl.js

```diff
@@ -74,8 +74,18 @@ angular.module("Prometheus.controllers")
     $scope.nextCycleRedraw();
   };
 
+  $window.history.replaceState({initial: true}, "Promdash - " + dashboardName, $window.location.href);
```

Comment:
I might be wrong, but isn't that a different title we usually use? It seems the title set by Rails is "<dashboard> - Prometheus Dashboard". Can we read the current value here, that would avoid us from maintaining the title format in two different places.